web/src/tippyjs.js

```diff
@@ -78,6 +112,16 @@ const LONG_HOVER_DELAY = [750, 20];
 // distracting users unnecessarily.
 const EXTRA_LONG_HOVER_DELAY = [1500, 20];
 
+function message_list_tooltip(target, props) {
+    delegate("body", {
+        target,
+        delay: LONG_HOVER_DELAY,
```

Comment:
We shouldn't assume all message tooltips will have this `delay`, so let's move it back to where they were defined.